### PR TITLE
remove GOOGLE_ENABLE

### PR DESCRIPTION
**Pull request by briskt**
### Removed
- Removed `GOOGLE_ENABLE` option since we never use it.

https://itse.youtrack.cloud/issue/IDP-1041

### CODE REVIEW

File: docker-compose.yml [hunk 1]
```diff
@@ -298,14 +290,6 @@ services:
...
       # Utilize custom metadata
       - ./development/sp2-local/metadata/saml20-idp-remote.php:/data/vendor/simplesamlphp/simplesamlphp/metadata/saml20-idp-remote.php
-
-      # Local modules
-      - ./modules/mfa:/data/vendor/simplesamlphp/simplesamlphp/modules/mfa
-      - ./modules/expirychecker:/data/vendor/simplesamlphp/simplesamlphp/modules/expirychecker
-      - ./modules/profilereview:/data/vendor/simplesamlphp/simplesamlphp/modules/profilereview
-      - ./modules/silauth:/data/vendor/simplesamlphp/simplesamlphp/modules/silauth
-      - ./modules/sildisco:/data/vendor/simplesamlphp/simplesamlphp/modules/sildisco
-      - ./modules/material:/data/vendor/simplesamlphp/simplesamlphp/modules/material
```
Comment (mtompset) [inline docker-compose.yml:308]: Are you sure about these local modules being removed?

Comment (briskt) [reply docker-compose.yml:308]: Yes, I shouldn't have added them to begin with. The SP containers are pretty bare-bones. They are only there to receive login requests and redirect to the Hub and/or IdP containers. Our custom modules are all for IdP and Hub use.

**Review by hobbitronics — APPROVED** [review]
I'm not sure what enabling the "core", "saml", and "admin" modules does, but assume they won't break anything.

**Comment by briskt** [comment]
> I'm not sure what enabling the "core", "saml", and "admin" modules does, but assume they won't break anything.

The old way, in SimpleSAMLphp 1.x, was to include a file named `enable` or `default-enable` in the module directory. The new way, in SimpleSAMLphp 2.x, is to use this configuration. These three are the standard (non-custom) modules that we use.

**Pull request merged**